xml/2018-07-15-end-of-year-survey.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="2018 Member Survey"
+  agenda="false"
+  social="false"
+  logos="false">
```

Comment:
Perhaps we should include social and logos? We tend to only exclude them on mail shots etc. Default is true so you can just remove these attributes.